diff --git a/dist/index.cjs b/dist/index.cjs
index 14ee72e12556f74e3efa56003c9956025edde54b..3a573d0295cf56b09251672aace1452610a662c9 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -107,6 +107,9 @@ function demoBlockPlugin(md) {
         return defaultRender(tokens, idx, options, env, self);
       }
       const frontmatter = env.frontmatter;
+      if(!(frontmatter.realPath ?? path$1)) {
+        return defaultRender(tokens, idx, options, env, self);
+      }
       const mdDir = path.dirname(frontmatter.realPath ?? path$1);
       const srcPath = path.resolve(mdDir, props.src);
       const code = fsExtra__default.readFileSync(srcPath, "utf-8");